packages/react-charting/src/components/HorizontalBarChart/HorizontalBarChart.styles.ts

```diff
@@ -1,4 +1,4 @@
-import { FontWeights } from '@fluentui/react/lib/Styling';
+import { FontSizes, FontWeights } from '@fluentui/react/lib/Styling';
 import { IHorizontalBarChartStyleProps, IHorizontalBarChartStyles } from './HorizontalBarChart.types';
```

Comment:
Is the fix applied to stacked bar chart and multi stacked bar chart as well

Reply:
Will create a separate PR for that